WcaOnRails/app/views/registrations/_register_form.html.erb

```diff
@@ -0,0 +1,31 @@
+<% url = @registration.new_record? ? competition_registrations_path(@competition)
+                                   : registration_path(@registration) %>
+<%= horizontal_simple_form_for @registration, url: url, html: { class: 'are-you-sure no-submit-on-enter' } do |f| %>
```

Comment:
Since `current_user.can_edit_registration?(@registration)` is used 4 times, I'd break it out as a variable:

`edit_disabled = !current_user.can_edit_registration?(@registration)`